sdk/resources-subscriptions/arm-resources-subscriptions/review/arm-resources-subscriptions.api.md

```diff
@@ -71,10 +71,11 @@ export interface Location {
     readonly subscriptionId?: string;
     readonly type?: LocationType;
 }
+export { Location_2 as Location }
```

Comment:
This PR should probably not change what you are exporting. I think there are 2 `Location` getting exported accidentally.

Reply:
Please see https://github.com/Azure/azure-sdk-for-js/pull/17917#issuecomment-930465924 and https://github.com/Azure/azure-sdk-for-js/pull/17917#issue-1010274710 for reference. TL;DR - we aren't changing what we're exporting, rather API Extractor is _exposing_ any time we shadow a global type.

From the consumer's perspective there are no changes